spec/fixtures/hieradata/common.yaml

```diff
@@ -71,7 +71,7 @@ govuk_jenkins::config::github_web_uri: wibble
 govuk_jenkins::jobs::deploy_app::applications: *deployable_applications
 govuk_jenkins::jobs::deploy_app_downstream::applications: *deployable_applications
 govuk_jenkins::jobs::run_rake_task::applications: *deployable_applications
-govuk_jenkins::jobs::content_data_api_re_run::re_run_rake_etl_main_process_cron_schedule: '0 3 * * *'
+govuk_jenkins::jobs::content_data_api_re_run::re_run_rake_etl_main_process_cron_schedule: '0 7 * * *'
```

Comment:
In hindsight, this didn't work, because it was a change to the specfile rather than the actual config 🤦